lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +219,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        for div in article_bs.find_all('div', class_='items-list-inner__item-title switcher-title font_20'):
+            for link in div.select('a'):
+                url = link['href']
+        return self.url_pattern + url
```

Comment:
should i extract only one url? (it says that method returns url(singular)) i thought that i should collect all the articles from the seed url like:
url += self.url_patter + link['href'] + '\n'
and then return str full of all urls

Reply:
yes, this method should extract just one article url from one element given

you may call this method several times in a loop from find_articles method